### DISCUSSION TITLE
wslg install fail

### DISCUSSION THREAD

**Discussion by imagtek** [microsoft/wslg#204] [Q&A]
Running 'wsl --update', from W10 command prompt, as per wslg installation instructions, returns 'invalid commandline option'

W10/Home update says my w10 installation is up-to-date, but attempt to install wsl-graphics-support_x64 from github returns 'requires build 21362  Current 19042'

This is very confusing. So I guess it is back to 'wsl 1' until this clears up?

**Comment by onomatopellan** [comment]
You are running Windows 10 version 20H1 (build 19042). You need to update to an insider Dev build in order to run WSLg or just wait for the upcoming update to version 21H2 in October-November.

**Comment by imagtek** [comment]
I gradually figured that out. I'm a bit slow.
I'll just wait for the formal release. I can't risk my development 
environment to a system crash and WSL 1/ vcxsrv works good enough for 
coding.
Thank you for the information!
John Powers
